tests/test_contamination_tech_reps.nf.test.snap

```diff
@@ -61,36 +61,36 @@
             true,
             true,
             true,
-            "Clone1_N1_trimmed_mature.sorted.idxstats:md5,761f06609a8c9b2ac05fe1505bf7f490",
-            "Clone1_N1_trimmed_mature.sorted.stats:md5,3f6611b81a95197d78202409cd26da0f",
-            "Clone1_N3_trimmed_mature.sorted.idxstats:md5,3ac5cf1e4b6c950720e3d5f840da6ce5",
-            "Clone1_N1_trimmed_mature.sorted.flagstat:md5,fc0b11d7f453e01b1eb50696f1d559b8",
-            "Clone1_N1_mature.sorted.idxstats:md5,58739274531355400f0b3c0ab7d2a526",
-            "Clone1_N3_trimmed_mature.sorted.flagstat:md5,9a723b10c4cab5d7ade554ce0c6656ee",
-            "Clone1_N3_trimmed_mature.sorted.stats:md5,7fc00a9d6000c711e71fe1d73e9797dd",
-            "Clone1_N1_mature.sorted.stats:md5,5b24697674ef3f11768fd3c53628bc18",
-            "Clone1_N1_mature.sorted.flagstat:md5,79bed8efbd7bc99d7f1cc38e4c37d60b",
+            "Clone1_N1_trimmed_mature.sorted.idxstats:md5,8c2bbcde56baa2b314404b93a020567f",
+            "Clone1_N1_trimmed_mature.sorted.stats:md5,088ba6bc7ba16a7fe795735d1b8d6923",
+            "Clone1_N3_trimmed_mature.sorted.idxstats:md5,fc67d3c97ac30f5f0cd5d09fc7c9fe57",
+            "Clone1_N1_trimmed_mature.sorted.flagstat:md5,1d03810df60fc58875e17f3734e908ab",
+            "Clone1_N1_mature.sorted.idxstats:md5,cf28da83c1063c2425c7bae548554cc8",
+            "Clone1_N3_trimmed_mature.sorted.flagstat:md5,2087da12bea640440fbed79a681608d7",
+            "Clone1_N3_trimmed_mature.sorted.stats:md5,7a1a48332e7d2779bb41c3466665e8c3",
+            "Clone1_N1_mature.sorted.stats:md5,ee5a60d2d759e884062cdf427e4d47af",
+            "Clone1_N1_mature.sorted.flagstat:md5,d23318288c499fe20ffa88f0790296ae",
             true,
             true,
             true,
             true,
             true,
             true,
-            "Clone1_N3_trimmed_mature_hairpin.sorted.idxstats:md5,e41871a5a8d432ad16455ba4cece7170",
```

Comment:
Same for these, we need to know that these changes are not affecting the output of the pipeline in an unwanted way

Reply:
I found that these files are actually non-deterministic, disregard this comment